<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableEditor</class>
 <widget class="QMainWindow" name="tableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="rowsCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rows:</string>
    </property>
   </widget>
   <widget class="QLabel" name="columnCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Columns:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rowsCounter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="columnCounter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="create">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="paddingCounter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="cellPadding">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cell padding:</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tableName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
